api-reference-v2/api-reference/api-key/api-key--retrieve.mdx

```diff
@@ -1,3 +1,3 @@
 ---
-openapi: get /v2/api_keys/{key_id}
+openapi: get /v2/api_keys/{id}
```

Comment:
Should we consider using hyphens here instead, to keep things consistent:

```suggestion
openapi: get /v2/api-keys/{id}
```

CC: @jarnura @Narayanbhat166

Reply:
We'd be taking this up in a separate PR.